x-pack/legacy/plugins/apm/server/lib/settings/cm/get_service_names.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Setup } from '../../helpers/setup_request';
+import { PromiseReturnType } from '../../../../typings/common';
+import {
+  PROCESSOR_EVENT,
+  SERVICE_NAME
+} from '../../../../common/elasticsearch_fieldnames';
+
+export type CMServicesAPIResponse = PromiseReturnType<typeof getServiceNames>;
```

Comment:
Terminology: When I started this PR I was unsure about what terminology we were using. Settings? Central config management? And how was it abbreviated? So I might have muddied the waters from the get go.
Would be nice to get this cleaned up a bit.

In this case I see `CMServicesAPIResponse` where "CM" is a (bad) abbreviation of Central Config Management. Since we now call it Agent Remote Configuration perhaps this abbreviation is not so apt anymore.

Reply:
We actually landed on APM Agent Configuration as the feature name (https://github.com/elastic/apm-dev/issues/553) 🙂

Reply:
Great! I'll try to remember that and stop saying central configuration management :D